contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,127 @@
+---
+title: The most useful product health metrics
+date: 2023-02-28
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+category: Product growth
+tags:
+ - Guides
+ - Product metrics
+---
+
+Like the human body, a product is a complicated system you want to keep healthy. Measuring product health with metrics helps this happen. They help you understand what parts of your product are working as expected and what parts need interventions to make them healthy.
+
+This post goes over:
+
+1. what makes a good health metric
+2. common product health metrics
+3. how to choose the right product health metrics
+
+## What is a product health metric?
+
+If you think of a product like a human body, then product health metrics are like the vital signs. They are the body temperature, blood pressure, pulse rate, and respiration rate of a product. They are a way to constantly judge how your product is doing. They show how changes, updates, and the environment are impacting it.
+
+Product health metrics are usually steady. Radical changes are a bad sign. Having your temperature way above average means you are sick and need medicine. If a product health metric sees a radical change, it needs intervention.
+
+Companies aim to improve product health metrics steadily over the long term. Like how exercise improves your pulse and respiration rates, adding enhancements and fixing bugs improve your product health metrics.
+
+Because they are constantly measured and steadily improved, product health metrics generally update in real-time and changes are measured over a length of time. For example, a product health metric might be down for the week, which triggers a minor adjustment in focus for next week. It can at the same time be up for the month, which is an encouraging sign. 
+
+Another similarity to your body’s health is that mature products generally care more about them. Early-stage companies are searching for product fit. This often requires product changes or pivots that lead to radical changes in product health metrics. It doesn’t mean the product is "sick," it means they haven’t figured out what "healthy" means yet.
```

Comment:
This section is really good. The analogy works and I like how you explained the distinction between mature products and early-stage ones – maybe we should link to your 80/20 guide here?